SoftLayer/managers/storage_utils.py

```diff
@@ -1003,6 +1003,93 @@ def prepare_duplicate_order_object(manager, origin_volume, iops, tier,
     return duplicate_order
 
 
+def prepare_modify_order_object(manager, origin_volume, new_iops, new_tier,
+                                new_size, volume_type):
+    """Prepare the duplicate order to submit to SoftLayer_Product::placeOrder()
```

Comment:
This should be something like "modification order" instead of "duplicate order".